project_dev/readme/DESCRIPTION.rst

```diff
@@ -0,0 +1,5 @@
+This module adds a field "Devs" in project tasks with two goals:
+
+- Tracking which dev(s) are working on a specific task.
+- Restricting devs access only to tasks where they are assigned as devs,
```

Comment:
Restricting "devs" users visibility only to tasks where they are assigned as devs,

Reply:
done